### ISSUE TITLE
zoomeye now require apikey other than username and password

### ISSUE THREAD

**Issue by m4ra7h0n** [OWASP/Amass#1033]
<img width="1267" alt="image" src="https://github.com/user-attachments/assets/56fcbed3-b9ad-4ae4-b744-8b86df458914">
<img width="404" alt="image" src="https://github.com/user-attachments/assets/ee59c648-b47c-4e93-9091-1b7657ae62c6">

**Comment by sn1p3rt3s7** [comment]
@caffix
Looks genuine.
I will look into this issue.

**Comment by sn1p3rt3s7** [comment]
Now `/login` endpoint where username and password is given is not required
Instead the API key is passed 
```
"message": "This API has been deprecated. Please use -H API-KEY: xxxx for authentication."
```

I passed the API key as header as shown below.
Then I got credit insufficient error.
![Image](https://github.com/user-attachments/assets/99f8888f-8f78-4617-a7d4-ac9782a9cee3)

It seems registered users using free subscription needs to buy points
[Pricing](https://www.zoomeye.ai/pricing)
![Image](https://github.com/user-attachments/assets/f08a270e-01fb-4d70-b53b-cae59edc19b8)

[Points hub](https://www.zoomeye.ai/profile/record) in Settings 
There is `General point` of 3000 and `Zoom-Eye point` of zero for registered users.
Still they can buy Zoom-eye points from pricing section.
![Image](https://github.com/user-attachments/assets/b74349c8-ec01-44a4-9f69-b514248b13da)

In short, Zoomeye API feature is not free now.
